apps/api/src/modules/providers/sms-plivo/sms-plivo.service.ts

```diff
@@ -10,7 +10,25 @@ export interface SmsPlivoData {
 export interface SmsPlivoResponseData {
   api_id: string;
   message: string;
-  message_uuid: string[];
+  messageUuid: string;
+}
+
+export interface PlivoMessageStatusResponse {
+  api_id: string;
+  error_code: string;
+  from_number: string;
+  message_direction: string;
+  messageState: string;
```

Comment:
Why is this in camelCase when others are in snake_case?

Reply:
This should be camelCase for all properties (docs don't mention it for messaging, but this reference tells that Node.js uses camelCase: https://docs.plivo.com/docs/voice/api/response/

Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/261/commits/5ce4e67bd9877772054c24dac5a90f5ae9c00aad